docs/src/modules/components/AppNavDrawer.js

```diff
@@ -428,6 +435,7 @@ function AppNavDrawer(props) {
     const isProductScoped =
       router.asPath.startsWith('/x') ||
       router.asPath.startsWith('/material-ui') ||
+      router.asPath.startsWith('/joy-ui') ||
       (router.asPath.startsWith('/system') && FEATURE_TOGGLE.enable_system_scope) ||
       router.asPath.startsWith('/base');
```

Comment:
Oh interesting, so the current direction is:

- MUI X
- Material UI
- Joy UI
- MUI System
- MUI Base

Alright, sounds good 👍